core/lib/mcxtx.h

```diff
@@ -218,43 +218,51 @@
                                           rip/eip register. */
     };
     byte padding[PRE_XMM_PADDING]; /**< The padding to get zmm field 64-byte aligned. */
-    /**
-     * The SSE registers xmm0-xmm5 (-xmm15 on Linux) are volatile
-     * (caller-saved) for 64-bit and WOW64, and are actually zeroed out on
-     * Windows system calls.  These fields are ignored for 32-bit processes
-     * that are not WOW64, or if the underlying processor does not support
-     * SSE.  Use dr_mcontext_xmm_fields_valid() to determine whether the
-     * fields are valid.
-     *
-     * When the fields are valid, on processors with AVX enabled (i.e.,
-     * proc_has_feature(FEATURE_AVX) returns true), these fields will
-     * contain the full ymm register values; otherwise, the top 128
-     * bits of each slot will be undefined.
-     */
+    union {
```

Comment:
My previous comment "You can see what the existing code above is doing: please follow that lead" implied also using a doxygen comment on the union like all the others are doing!

Reply:
Done.